src/player.cpp

```diff
@@ -9539,7 +9539,7 @@ void player::do_read( item &book )
 
         if( book_fun_for( book, *learner ) != 0 ) {
             //Fun bonus is no longer calculated here.
-            learner->add_morale( MORALE_BOOK, 0, book_fun_for( book, *learner ) * 5, 6_minutes, 3_minutes, true,
+            learner->add_morale( MORALE_BOOK, book_fun_for( book, *learner ) * 5, book_fun_for( book, *learner ) * 5, 6_minutes, 3_minutes, true,
```

Comment:
Is it intentional that the max morale is *5 instead of *15?

Reply:
Nope. It sure isn't.